apps/cookbook/src/app/examples/default-list-example/default-list-example.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+const config = {
+  template: `<kirby-page
+></kirby-page>`,
+  styles: [
```

Comment:
Reverting margin (margin-top) on `<li>`s should be a part of the opt-out guidelines.

The declaration `margin-block-start: revert;` can be applied to the affected `<li>`s